src/DevOps/Command/GenerateOpenApi.php

```diff
@@ -36,6 +36,12 @@ class GenerateOpenApi extends Command
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        // Ensure plugin vendor autoloader is loaded for PayPal SDK classes
+        $vendorAutoload = self::ROOT_DIR . '/vendor/autoload.php';
```

Comment:
Any plugin with external dependencies will be installed as composer dependency of platform. We do not expect the vendor directory present in the plugin on purpose